libevmasm/Instruction.cpp

```diff
@@ -295,6 +296,7 @@ static const std::map<Instruction, InstructionInfo> c_instructionInfo =
 	{ Instruction::CALLCODE,	{ "CALLCODE",		0, 7, 1, true, Tier::Special } },
 	{ Instruction::RETURN,		{ "RETURN",			0, 2, 0, true, Tier::Zero } },
 	{ Instruction::DELEGATECALL,	{ "DELEGATECALL",	0, 6, 1, true, Tier::Special } },
+	{ Instruction::CREATE2,		{ "CREATE2",		0, 2, 1, true, Tier::Special } },
```

Comment:
I asked this on the EIP and not sure whether it removes the `value` parameter or not.